packages/shorebird_cli/lib/src/commands/release/ios_releaser.dart

```diff
@@ -42,6 +42,17 @@ class IosReleaser extends Releaser {
   @override
   Future<void> assertArgsAreValid() async {
     argResults.assertAbsentOrValidPublicKey();
+
+    if (argResults.wasParsed('release-version')) {
+      logger.err(
+        '''
```

Comment:
Should this text go in the CommandArguments for the sake of dry?

Maybe adding a new field to the CommonArgument to store it, like `wrongUsageHelp` (or similar).

Then on both places you could just `logger.err(CommonArguments.releaseVersion.wrongUsageHelp)`.

Reply:
I think the instinct is good, but both the help message and error messages are command-specific.